Makefile.am

```diff
@@ -3,7 +3,7 @@ ACLOCAL_AMFLAGS = -I m4
 
 UAP_REGEX_FILE=regexes.yaml
 
-SUBDIRS=ext docs docker
```

Comment:
Yes agreed this is definitely a bad idea :) I've changed with the latest commit to remove it entirely, so that docker must be built explicitly. The docker stuff is mainly for regression in travis, so that seems like the best approach.